docs/ldml/tr35-keyboards.md

```diff
@@ -1314,11 +1322,18 @@ _Attribute:_ `form` (required)
 > This attribute specifies the physical layout of a hardware keyboard,
 > or that the form is a `touch` layout.
 >
-> It is recommended to always have at least one hardware (non-touch) form.
+> When using an on-screen keyboard, if there is not a `<layers form="touch">`
+> element, the hardware elements can be used for on-screen use. However, a `touch` keyboard
+> may not be used with a hardware keyboard.
+>
```

Comment:
```suggestion
> When using an on-screen keyboard, if there is not a `<layers form="touch">`
> element, the hardware elements can be used for on-screen use. 
```

there's already text below that says that the implementation may need to choose a different file. Doesn't make a statement about conformance.